rebabel_format/converters/nlp_part_of_speech.py

```diff
@@ -35,7 +36,7 @@ def process_line(self, line: str):
         line -- a word and its part of speech separated by a delimiter (ex: jumped/VERB)
         """
         split_line = line.strip().split(" ")
-        delimiter = "/"
+        delimiter = Parameter(default="/", type=str)
```

Comment:
this should be outside the function, and then inside the function you can refer to `self.delimiter`